pocs/camera/camera.py

```diff
@@ -189,15 +208,59 @@ def filter_type(self):
         else:
             return self._filter_type
 
+    @property
+    def is_cooled_camera(self):
+        """ True if camera has image sensor cooling capability """
+        return self._is_cooled_camera
+
+    @property
+    def is_temperature_stable(self):
+        """ True if image sensor temperature is stable, False if not.
+
+        An uncooled camera, or cooled camera with cooling disabled, will always return False.
+        """
+        if self.is_cooled_camera and self.cooling_enabled:
+            at_target = abs(self.temperature - self.target_temperature) \
```

Comment:
Probably overkill for this PR but does it make sense to have some kind of moving average for the "current" temperature. As far as I can tell this is only checking for momentary stability (which is probably find in most cases).

Reply:
The assumption being made is that if it's at the right temperature now and isn't having to use 100% cooling power to get there then the camera's temperature control system should be able to maintain the correct temperature, at least until the end of the next exposure. I think that will generally be true, unless the ambient temperature increases rapidly.

Actually measuring the stability with a series of temperature readings would be better than just making assumptions about it, but probably not a priority.